app/views/foods/index.html.erb

```diff
@@ -0,0 +1,26 @@
+<%= stylesheet_link_tag "application"%>
+
+<div class="logger_in_form">
+<p><%= notice %></p>
+
+<h1 class="custom-login">Listing foods</h1>
+<h2 class="custom-login">My Favourite Foods</h2>
+
+<%= stylesheet_link_tag "application"%>
+
+<div class="logger_in_form">
+<div id="foods">
+  <div class="food">
+    <p><strong>Food ID: </strong></p>
+    <p><strong>Name: </strong></p>
+    <p><strong>Measurement Unit: </strong></p>
+    <p><strong>Price: </strong></p>
+    <p><strong>Quantity: </strong></p>
+    <p><strong>User ID: </strong></p>
+    <p><stron>Activity: </strong></p>
+  </div>
+  <%= @foods.each do |food| %>
+    <%= render food %>
+  <% end %>
+</div>
```

Comment:
- [ ]  Please add a button or link that leads to a form that allows users to add new food.
- [ ] Kindly make this view matches the given [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/recipe-app/images/food_index.png?raw=true).

Reply:
Thank you for this